openvino_tensorflow/backend_manager.cc

```diff
@@ -34,11 +35,20 @@ Status BackendManager::SetBackend(const string& backend_name) {
   m_backend = backend;
   if (bname.find("MYRIAD") != string::npos) {
     m_backend_name = "MYRIAD";
+    m_tf_frontend_disabled = true;
   } else if (bname.find("GPU") != string::npos) {
     m_backend_name = "GPU";
   } else {
     m_backend_name = bname;
+    if (bname.find("HDDL") != string::npos) m_tf_frontend_disabled = true;
   }
+
+  if (std::getenv("OPENVINO_TF_DISABLE_TFFE") != nullptr) {
+    if (1 == std::stoi(std::getenv("OPENVINO_TF_DISABLE_TFFE"))) {
+      m_tf_frontend_disabled = true;
+    }
+  }
+
```

Comment:
I think these changes are already done in TF FE branch. Why do we need these here as well?